Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiBase.cs

```diff
@@ -281,23 +285,58 @@ public ObservableList<OptionalValue> GetListOfParamEnums(SwaggerParameter swagge
                         {
                             Value = item.ToString(),
                             ItemName = item.ToString(),
+                            IsDefault = true
                         };
                         lstOptions.Add(value);
                     }
                 }
-                if (swaggerParameter.Enumeration != null && swaggerParameter.Enumeration.Count > 0)
+                else if (swaggerParameter.Enumeration != null && swaggerParameter.Enumeration.Count > 0)
                 {
                     foreach (object item in swaggerParameter.Enumeration)
                     {
                         OptionalValue value = new OptionalValue()
                         {
                             Value = item.ToString(),
                             ItemName = item.ToString(),
+                            IsDefault = true
                         };
                         lstOptions.Add(value);
                     }
 
                 }
+                else if (swaggerParameter.ActualSchema.ActualProperties.Count>0 || swaggerParameter.ActualSchema.Enumeration.Count>0)
```

Comment:
Same condition is being checked in below if blocks, please verify if two conditions are needed

Reply:
removed, it was not required in else if block.